### PR TITLE
Feat/create select

### PR DESCRIPTION
**Pull request by atomictangerine**
## What I did

* add select component that extends Radix select
* add documentation
* add examples
* add storybook example

### CODE REVIEW

File: ui/kit/package.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
     "@washingtonpost/wpds-pagination-dots": "*",
     "@washingtonpost/wpds-radio-group": "*",
     "@washingtonpost/wpds-scrim": "*",
+    "@washingtonpost/wpds-select": "0.19.0",
```
Comment (atomictangerine) [inline ui/kit/package.json:90]: just noticed this guy. will be updating

File: ui/select/src/SelectContent.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import { SelectContext } from "./Select";
+
+import { styled, theme } from "@washingtonpost/wpds-theme";
+import { Icon } from "@washingtonpost/wpds-icon";
+import { ChevronUp, ChevronDown } from "@washingtonpost/wpds-assets";
+
+import * as SelectPrimitive from "@radix-ui/react-select";
+import type * as WPDS from "@washingtonpost/wpds-theme";
+
+const StyledContent = styled(SelectPrimitive.Content, {
```
Comment (wp-aberg) [inline ui/select/src/SelectContent.tsx:11]: It looks like the Figma design includes a border style for the content
` border: `1px solid ${theme.colors.gray300}` `

File: ui/select/src/SelectContent.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  backgroundColor: theme.colors.secondary,
+  color: theme.colors.primary,
+  fontFamily: theme.fonts.meta,
+  fontSize: theme.fontSizes["100"],
+  fontWeight: theme.fontWeights.light,
+  lineHeight: theme.lineHeights["125"],
+});
```
Comment (wp-aberg) [inline ui/select/src/SelectContent.tsx:18]: I know this is brittle, but it just looks weird to have the content offset, can we try 
`transform: "translate(22px, 9px)",` in StyledContent?

After:
image

File: ui/select/src/SelectContent.tsx (hunk 2 above)
Comment (atomictangerine) [reply ui/select/src/SelectContent.tsx:11]: ahh good catch

File: ui/select/src/SelectContent.tsx (hunk 3 above)
Comment (atomictangerine) [reply ui/select/src/SelectContent.tsx:18]: yes, I agree. It could be brittle, but it should work for most use cases :) I'll try that

**Review by wp-aberg — APPROVED** [review]
All set when you are

**Pull request merged**